chia/wallet/puzzles/dao_proposal.clvm

```diff
@@ -307,10 +310,12 @@
               )
               (if
                 (all
+                  ; COMMENT: If this is used for both things, the function could use a different name probably
                   (> TOTAL_VOTES (calculate_win_percentage vote_coin_id_or_current_cat_issuance lockup_innerpuzhash_or_attendance_required))
                   (> YES_VOTES (calculate_win_percentage TOTAL_VOTES previous_votes_or_pass_margin))
                 )
                 (a INNERPUZ vote_amount_or_solution)  ; run_chialisp: run proposed chialisp ; XXX Matt, Geoff: should we also re-create ourself with previous_votes=0 in this case?
+                ; COMMENT: Is there even a reason to ping the treasury in this case? Nothing happens on it, it can just stay the same right?
```

Comment:
we still need the values that the treasury stores about things like "proposal length" and "pass requirements"